src/main/webapp/i18n/en/guidedTourCourseOverview.json

```diff
@@ -48,8 +48,8 @@
             },
             "contact": {
                 "headline": "Contact the Artemis team",
-                "content": "Feel free to contact us to leave us some feedback, request features or to report bugs in the application. If you're motivated and want to contribute to Artemis, then head over to our Github and submit your first Pull Requests",
-                "link": "Visit the Artemis Github page"
+                "content": "Feel free to contact us to leave us some feedback, request features or to report bugs in the application.",
```

Comment:
```suggestion
                "content": "Feel free to contact us, to leave us some feedback, request features or to report bugs in the application.",
```

Reply:
@maxr96 The intention of the sentence is that the user can contact the Artemis team in order to leave some feedback, so the comma is unnecessary imo.